my-app/angular.json

```diff
@@ -0,0 +1,128 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "my-app": {
+      "projectType": "application",
+      "schematics": {
+        "@schematics/angular:component": {
```

Comment:
[suggestion] would be nice to set default CD strategy to OnPush, you can do it with ng config (https://angular.io/cli/config) command or manually.